config.toml

```diff
@@ -1,97 +1,60 @@
-    target = "assets/sass/bulma"
-
-  [[module.imports]]
-    path="github.com/twbs/bootstrap"
-  [[module.imports.mounts]]
-    source = "dist/css"
-    target = "assets/css/bootstrap"
-  [[module.imports.mounts]]
-    source = "dist/js"
-    target = "static/js/bootstrap"
-
-  [[module.imports]]
-    path="github.com/jquery/jquery"
-  [[module.imports.mounts]]
-    source = "dist"
-    target = "static/js/jquery"
-
-  [[module.imports]]
-    path="github.com/mermaid-js/mermaid"
-  [[module.imports.mounts]]
-    source = "dist"
-    target = "assets/js/mermaid"
-
-# Syntax highlighting settings
-[markup]
-  [markup.goldmark]
-    [markup.goldmark.renderer]
-      unsafe = true
-    [markup.goldmark.parser]
-      [markup.goldmark.parser.attribute]
-        block = true
-        title = true
-  [markup.highlight]
-    lineNos = true
-    lineNumbersInTable = true
-    noClasses = false
-    style = 'monokailight'
-  [markup.tableOfContents]
-    endLevel = 3
-    ordered = false
-    startLevel = 1
-
-[params]
-description = 'Automatic and Verifiable Artifact Resolution'
-images = [ '/img/logos/omnibor-logo-stacked.png' ]
-subtitle = 'Automatic and Verifiable Artifact Resolution'
-fontawesomeversion = "5.8.1"
-[[params.fonts]]
-name = "Nunito Sans"
-sizes = [300, 400, 600, 700]
-type = "sans_serif"
-
-[[params.fonts]]
-name = "Roboto Mono"
-sizes = [300, 400, 600, 700]
-type = "monospace"
-
-[params.social]
-    github = "https://github.com/omnibor/"
-    twitter = "https://twitter.com/omnibor"
-
-[params.logos]
-navbar = "omnibor-logo.svg"
-hero = "omnibor-logo.svg"
-
-[menu]
-[[menu.navbar]]
-name = "Resources"
-url = "/resources/"
-weight = 1
-[[menu.navbar]]
-name = "Community"
-url = "/community/"
-weight = 2
-[[menu.navbar]]
-name = "Glossary"
-url = "/glossary/"
-weight = 3
\ No newline at end of file
+base_url = "https://omnibor.io"
+title = "OmniBOR"
+compile_sass = false
+build_search_index = true
+
+[markdown]
+
+highlight_code = true
+highlight_theme = "ayu-light"
+
+[extra]
+
+nav = [
+    { name = "Spec", url = "@/spec/_index.md", pill = "v0.1" },
+    { name = "Docs", url = "@/docs/_index.md" },
+    { sep = true },
+    { name = "Blog", url = "@/blog/_index.md" },
+    { name = "Project", url = "@/project.md" },
+    { name = "Contribute", url = "@/contribute.md" },
+    { sep = true },
+    { icon = "github", url = "https://github.com/omnibor" },
+]
+
+footer = [
+    [
+        { name = "Spec", title = true },
+        { name = "Stable", url = "@/spec/v0.1/_index.md", pill = "v0.1" },
+        { name = "Latest", url = "https://github.com/omnibor/spec", external = true },
+    ],
+    [
+        { name = "Tools", title = true },
+        { name = "OmniBOR CLI", url = "https://github.com/omnibor/omnibor-rs/tree/main/omnibor-cli", external = true },
+        { name = "bomsh", url = "https://github.com/omnibor/bomsh", external = true },
+        { name = "jbor", url = "https://github.com/omnibor/jbor", external = true },
```

Comment:
What became of llvm clang, gcc, binutils, and patch here?

Reply:
Added!